bcda/api.go

```diff
@@ -153,6 +175,11 @@ func bulkRequest(t string, w http.ResponseWriter, r *http.Request) {
 	}
 	defer rows.Close()
 	var id string
+	var jobCount = 0
+	maxBeneficiaries, err := strconv.Atoi(os.Getenv("BCDA_FHIR_MAX_RECORDS"))
```

Comment:
You can use `getEnvInt` to automatically set the default if the env var doesn't exist.  
Here is an example: https://github.com/CMSgov/bcda-app/blob/master/bcda/api.go#L534

Reply:
great.  I had no idea this existed.